### ISSUE TITLE
How to create topic?

### ISSUE THREAD

**Issue by DayTimeDreamer** [fechanique/cordova-plugin-fcm#484]
Hi, how to create topic? It says the topic can't be found when I send notification by topics on the console.

**Comment by Gerben321** [comment]
When you subscribe to a non-existing topic it's made automatically, according to the Firebase docs.

**Comment by DayTimeDreamer** [comment]
Yes, the Firebase docs do say.  But I fail to create topics and I've waited for two days. Then we have to use the token to create topic on the server side.
